src/Common/src/Interop/Unix/System.Native/Interop.GetHostName.cs

```diff
@@ -27,11 +25,10 @@ internal static unsafe string gethostname()
                 // which should only happen if the buffer we supply isn't big
                 // enough, and we're using a buffer size that the man page
                 // says is the max for POSIX (and larger than the max for Linux).
-                Debug.Fail("gethostname failed");
+                System.Diagnostics.Debug.Fail("gethostname failed");
```

Comment:
Ah sorry this was a new file and I copy+paste the header and using's from a template. Will add the using back to keep it a simpler change to track